src/FileObject.php

```diff
@@ -10,6 +10,13 @@
  */
 class FileObject
 {
+    /**
+     * Default content type.
+     *
+     * @var string
+     */
+    public static $defaultContentType = 'application/octet-stream';
```

Comment:
This might well be set as a constant:

```suggestion
    public const DEFAULT_CONTENT_TYPE = 'application/octet-stream';
```